features/snippet.feature

```diff
@@ -0,0 +1,7 @@
+Feature: Snippet life cycle
+
+    Scenario: Creating a snippet
+        Given I am authenticated
+        When I fill in the snippet creation form
```

Comment:
For that purposes can recommend you this vim plugin https://github.com/tpope/vim-cucumber (it also includes some bonus stuff).